app/models/cp/benchmark.rb

```diff
@@ -2,25 +2,41 @@
 #
 # Table name: cp_benchmarks
 #
-#  id         :bigint           not null, primary key
-#  sector_id  :bigint
-#  date       :date             not null
-#  benchmarks :jsonb
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
+#  id           :bigint           not null, primary key
+#  sector_id    :bigint
+#  release_date :date             not null
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  emissions    :jsonb
+#  scenario     :string
 #
 
 module CP
   class Benchmark < ApplicationRecord
     belongs_to :sector
 
-    scope :latest_first, -> { order(date: :desc) }
-    scope :by_date, -> { order(:date) }
+    scope :latest_first, -> { order(release_date: :desc) }
+    scope :by_release_date, -> { order(:release_date) }
 
-    validates_presence_of :date
+    validates_presence_of :release_date, :scenario
```

Comment:
Maybe we should validate `scenario` with inclusion on the known scenario list (https://github.com/Vizzuality/laws_and_pathways/pull/57/files#diff-bcd1663b967ed112eee68c615f3ac627R4)?

@simaob do you know if that list (of scenarios) should be open or rather strict? Do we need to manage this list as well?

Reply:
I would keep that as just a name with the flexibility to change to whatever. They already have different names for different release dates and sectors. As long we don't need any complex reports it should be fine.

Reply:
hmm but later we will need only a known list of scenario on the frontend. How will we map it if there are more scenario types in the DB?

![Zrzut ekranu 2019-09-18 o 12 17 42](https://user-images.githubusercontent.com/10665/65140194-56abae80-da0e-11e9-853a-87a7689a2cc2.png)

Reply:
You only need names, and those benchmarks need to be grouped by `release_date` and only one benchmark group should be taken (everything depends on assessment date - an appropriate benchmark should be taken by date). Scenario names will be from that chosen benchmark group.

Reply:
Not sure if the colour has any meaning, I can try to double check with Fausto. If not we can have a number of colours that are flexible enough to an expected number of scenarios and then sort them always the same way, so that they always get the same colour. Something like that?

Reply:
We can show data against only one benchmark set/group.

Reply:
If the color has meaning from the lowest to the highest scenario we can add colors based on the average of emission values, scenario name won't matter that much.

Reply:
oh right, good point!

Reply:
@kowal is right here, on the image below I can only see the top of the chart, but looks like that chart needs standardized scenario names across different sectors. They are not standardized now, so I believe in the next iteration when we know that all scenarios are the same we can add validation to be only those from strict list.